pkg/proto/sqlflow.proto

```diff
@@ -67,8 +67,9 @@ message FetchResponse {
         SUCCEEDED = 2;
         FAILED = 3;
         ERROR = 4;
+        SKIPPED = 5;
```

Comment:
Do we really need the `SKIPPED` phase, I don't know what does the client do for the skipped phase.

Reply:
According to the Argo source code, the client should consider SKIPPED as the completion phase. I suspect skip is only used for nodes, never for the workflow. But I feel it is safer to follow the Argo source.

https://github.com/argoproj/argo/blob/99db30d67b42cbd9c7fa35bbdd35a57040c2f222/pkg/apis/workflow/v1alpha1/workflow_types.go#L841-L846